src/services/user/services/UsersModelService.js

```diff
@@ -28,6 +28,8 @@ const userModelService = feathersMongooseService({
 	lean: {
 		virtuals: true,
 	},
+	multi: true,
```

Comment:
what are differences of `LEAN:TRUE` and `LEAN: {VIRTUALS:TRUE}`? can we define one default?